### PR TITLE
fix: fix category link issue & add error handling

### PR DESCRIPTION
**Pull request by duidae**
fix
[[defect] 點擊文章頁 meta data 的分類/子分類時，無法正確跳轉到對應頁面](https://app.asana.com/0/1202961977949746/1203964075873151/f)
[[defect] keystone 設定的分類組合未完整顯示於文章頁](https://app.asana.com/0/1202961977949746/1203964075873152/f)

### CODE REVIEW

File: packages/react-article-components/src/components/aside/metadata.js [hunk 1]
```diff
@@ -293,14 +294,15 @@ const CategorySet = props => {
         </CategorySetFlex>
       )
     }
-    return (
+    const categoryPath = GET_CATEGORY_PATH_FROM_ID(set?.category?.id)
```
Comment (Aylie-Chou) [inline packages/react-article-components/src/components/aside/metadata.js:297]: The returned `post.category_set` data from go-api whould fetch category & subcategory name.
If there's not other consideration,  I think we could directly use it.

Comment (duidae) [reply packages/react-article-components/src/components/aside/metadata.js:297]: it has name but name is `人權司法`, not path, 

and I checked neither props do not include `path`
Screen Shot 2023-02-16 at 2 10 45 PM

nor api
Screen Shot 2023-02-16 at 2 12 04 PM

Comment (Aylie-Chou) [reply packages/react-article-components/src/components/aside/metadata.js:297]: yes, the api returned `name` value storing in database.
`humanrights` is used for fe data mapping.

Comment (Aylie-Chou) [reply packages/react-article-components/src/components/aside/metadata.js:297]: I found myself misunderstanding 😂
We need path here for the url not wording !

**Review by Aylie-Chou — APPROVED** [review]
LGTM 🙇

**Pull request merged**